src/client/src/apps/SpotlightRoute/Spotlight.tsx

```diff
@@ -112,11 +113,17 @@ export const Spotlight: FC = () => {
             <AdaptiveLoader size={14} speed={0.8} seperation={1.5} type="secondary"/>
             <Contacting>Contacting…</Contacting>
           </>
-        ) : (
-          <Suggestion onClick={() => handleIntent(response, platform)}>
-            {mapIntent(response)}
-          </Suggestion>
-        )}
+        ) : response ? (
+          <>
+            <Suggestion onClick={() => handleIntent(response, platform)}>
+              {mapIntent(response)}
+            </Suggestion>
+            <Suggestion>
+              {isSpotQuoteIntent(response) ?
+                <InlineQuote currencyPair={response.queryResult.parameters.fields.CurrencyPairs.stringValue}/> : null}
```

Comment:
sugegstion: use `getCurrencyPair` from `handleIntent`

Reply:
what if clicking on it would open a ticket too? (just call the related function from `handleIntent`)?

Reply:
Sure thing